client/component-lab/Notification/NotificationView.coffee

```diff
@@ -0,0 +1,208 @@
+_ = require 'lodash'
+React = require 'app/react'
+ReactDOM = require 'react-dom'
+kd = require 'kd'
+
+getVendorTransition = require 'app/util/getVendorTransition'
+
+Button = require 'lab/Button'
+Box = require 'lab/Box'
+
+styles = require './Notification.stylus'
+Helpers = require './helpers'
+Constants = require './constants'
+
+
+module.exports = class NotificationView extends React.Component
+
+    constructor: (props) ->
+
+      super props
+      @style = { zIndex: 100 + @props.index }
+      @notificationTimer = null
+      @isMounted = no
```

Comment:
`@isMounted` is a special function in Reactjs. It throws some warning, [check from here](https://facebook.github.io/react/blog/2015/12/16/ismounted-antipattern.html)